specification/gedcom.md

```diff
@@ -2142,16 +2142,42 @@ Substructures provide additional information about how that source applies to th
 - `QUAY`: an estimation of the reliability of the source in regard to these claims.
 - `MULTIMEDIA_LINK`: digital copies of the cited part of the source
 
-When no source record is available, a `voidPtr` and accompanying `NOTE` can be used to describe the source.
+When no source record is available, a `voidPtr` and accompanying `NOTE` or [documented extension](#extension-tags) `g7:PHRASE` can be used to describe the source.
+
+:::note
+When 7.0 was released it was decided to not document a SOUR.PHRASE structure and instead encode `SOURCE_CITATIONS` with no corresponding `SOURCE_RECORD` using a `NOTE`, with notes about the citation following, as follows:
```

Comment:
This statement seems inconsistent with https://gedcom.io/migrate/#non-pointer-sour-substructures which says that any 5.5.1 SOURCE_CITATIONS with no SOURCE_RECORD should be converted to SOURCE_CITATIONS with SOURCE_RECORDs, in which case this problem does not arise.

Reply:
See https://github.com/FamilySearch/GEDCOM/issues/97#issuecomment-1004414048 for more discussion